Integrations/integration-AWS-S3.yml

```diff
@@ -0,0 +1,289 @@
+
+
+    # The command demisto.command() holds the command sent from the user.
+    if demisto.command() == 'test-module':
+        response = s3.list_buckets()
+        if response['ResponseMetadata']['HTTPStatusCode'] == 200:
+          result = ('ok')
+          
+    if demisto.command() == 'aws-s3-create-bucket':
+        result = create_bucket(demisto.args())
+
+    if demisto.command() == 'aws-s3-delete-bucket':
+        result = delete_bucket(demisto.args())
+
+    if demisto.command() == 'aws-s3-list-buckets':
+        result = list_buckets(demisto.args())
+
+    if demisto.command() == 'aws-s3-get-bucket-policy':
+        result = get_bucket_policy(demisto.args())
+
+    if demisto.command() == 'aws-s3-set-bucket-policy':
+        result = set_bucket_policy(demisto.args())
+
+    if demisto.command() == 'aws-s3-delete-bucket-policy':
+        result = delete_bucket_policy(demisto.args())
+
+    if demisto.command() == 'aws-s3-download-file':
+        result = download_file(demisto.args())
+
+    if demisto.command() == 'aws-s3-delete-bucket-policy':
+        result = delete_bucket_policy(demisto.args())
+
+    if demisto.command() == 'aws-s3-list-bucket-objects':
+        result = list_objects(demisto.args())
+
+    demisto.results(result)
+    sys.exit(0)
+  type: python
+  commands:
+  - name: aws-s3-create-bucket
+    arguments:
+    - name: bucketName
+      required: true
+      description: The name of S3 bucket to create
+    - name: acl
+      required: true
+      auto: PREDEFINED
+      predefined:
+      - private
+      - public-read
+      - public-read-write
+      - authenticated-read
+      description: ACL for S3 bucket
+      defaultValue: private
+    - name: awsRegion
+      required: true
+      description: AWS Region to create bucket (optional, e.g us-west-1) if no region
+        is selected the default region will be used
+    outputs:
+    - contextPath: AWS.S3.NewBucket.BucketName
+      description: The name of the bucket that was created
+      type: string
+    - contextPath: AWS.S3.NewBucket.AWSRegion
+      description: 'The AWS Region the bucket was created '
+      type: string
+    description: Create AWS S3 bucket
+  - name: aws-s3-delete-bucket
+    arguments:
+    - name: bucketName
+      required: true
+      description: Name of S3 bucket to delete
+      isArray: true
+    outputs:
+    - contextPath: AWS.S3.DeletedBuckets.BucketName
+      description: Name of S3 bucket that was deleted
+      type: string
+    description: Delete AWS S3 bucket
+  - name: aws-s3-list-buckets
+    arguments: []
+    outputs:
+    - contextPath: AWS.S3.BucketList.BucketName
+      description: The name of the AWS S3 bucket
+    description: List all S3 buckets in AWS account
+  - name: aws-s3-get-bucket-policy
+    arguments:
+    - name: bucketName
+      required: true
+      description: Name of bucket
+      isArray: true
+    outputs:
+    - contextPath: AWS.S3.Policy.Version
```

Comment:
I think maybe the policy object should be contained in Bucket object, like `AWS.S3.Bucket.Policy....`

Reply:
fixed